tools/api-markdown-documenter/src/api-item-transforms/TsdocNodeTransforms.ts

```diff
@@ -568,30 +641,43 @@ function parseContentAsBlock(nodes: PhrasingContent[]): (ParagraphNode | ListNod
 	// This means that all adjacent list items are treated as separate items in a single root list.
 	// This is sufficient for our needs at the moment, but in the future we should add support for parsing nested lists.
 
-	const result: (ParagraphNode | ListNode)[] = [];
+	const result: (Paragraph | List)[] = [];
 	let iOutput = 0;
 	while (iOutput < outputLines.length) {
 		const current = outputLines[iOutput];
 
 		switch (current.type) {
 			case "paragraphLine": {
-				result.push(new ParagraphNode(current.content));
+				result.push({ type: "paragraph", children: current.content });
 				iOutput++;
 				break;
 			}
 			case "orderedListItem":
 			case "unorderedListItem": {
-				const items: ListItemNode[] = [];
+				const items: ListItem[] = [];
 				while (
 					iOutput < outputLines.length &&
 					outputLines[iOutput].type === current.type &&
 					(outputLines[iOutput] as ParsedOrderedListItem | ParsedUnorderedListItem)
 						.delimiter === current.delimiter
 				) {
-					items.push(new ListItemNode(combineAdjacentPlainText(outputLines[iOutput].content)));
+					items.push({
+						type: "listItem",
+						children: [
+							{
+								type: "paragraph",
```

Comment:
Note for reviewers: List items in `mdast` wrap block content, rather than phrasing content. Canonically, phrasing content is parented under a single paragraph under the list item node.